config/bidderinfo.go

```diff
@@ -229,8 +238,8 @@ func processBidderInfos(reader InfoReader, normalizeBidderName func(string) (ope
 		return nil, fmt.Errorf("error loading bidders data")
 	}
 
-	infos := BidderInfos{}
-
+	bidderInfos := BidderInfos{}
+	aliasInfos := aliasInfos{} //maintains map of bidderName to nullable fields in BidderInfo struct
```

Comment:
how about changing comment to following or something similar

```
// aliasInfos maintains a map of bidderName to nullable fields
// will be used to determine values that are not defined in the bidderInfo yaml
aliasInfos := aliasInfos{} 
```

Reply:
Why the use of the `aliasInfos` type alias? There's no need to copy `BidderInfos` in this regard. Might be easier to include the type directly. The name of `aliasNillableFieldsByBidder` might remove the need for a comment entirely.